posix/unix.c

```diff
@@ -182,7 +182,9 @@ static unixsock_t *unixsock_alloc(unsigned *id, int type, int nonblock)
 	proc_lockInit(&r->lock, &proc_lockAttrDefault, "unix.socket");
 
 	r->id = *id;
-	r->refs = 1;
+	/* alloc new socket with 2 refs: first ref for presence in the tree, second
```

Comment:
Maybe expand the comment to:
"one reference is for the socket's presence in the tree, and the second is for handling by the caller before returning the socket to the user (to protect against accidental removal by someone else)"